coreapi/main_api.py

```diff
@@ -10,50 +10,54 @@
 
 
-def FaceExp(cropped_face):
+def faceexp(cropped_face):
     """     Facical Expression Recognition of faces in image
-
     Args:
             *   cropped_face: numpy array of cropped face
-
     Workflow:
-            *   A numpy array of a cropped face is taken as input (RGB). inference input dimension requires
-                a dimension of (1,100,100,3) therefore the RGB input is first converted to grayscale image followed by
-                normalizing and resizing the required input dimension.
-
-            *   Now the processed output is further processed to make it a json format which is compatible to TensorFlow
-                Serving input.
-
-            *   Then a http post request is made at localhost:8501 . The post request contain data and headers.
-
+            *   A numpy array of a cropped face is taken as input (RGB).
+                inference input dimension requires dimension of (1,100,100,3)
+                therefore the RGB input is first converted to grayscale image
+                followed by normalizing & resizing to required input dimension.
+            *   Now the processed output is further processed to make it a
+                json format which is compatible to TensorFlow Serving input.
+            *   Then a http post request is made at localhost:8501.
+                The post request contain data and headers.
             *   Incase of any exception, it return empty string.
-
-            *   output from TensorFlow Serving is then parsed and a dictionary is defined which keeps the facial expression
-                name as key and prediction's output as value. The prediction values are floating point values which tells the
-                probability of the particular facial expression.
-
+            *   output from TensorFlow Serving is then parsed and a
+                dictionary is defined which keeps the facial expression name
+                as key and prediction's output as value. The prediction values
+                are floating point values which tells the probability of the
+                particular facial expression.
     Returns:
-            *   Dictionary having all the faces and corresponding facial expression and it's values.
+            *   Dictionary having all the faces and corresponding facial
+                expression and it's values.
     """
     img = cv2.resize(cropped_face, (100, 100), interpolation=cv2.INTER_AREA)
     img = np.array(img).reshape((1, 100, 100, 3))
     img = img / 255
-    data = json.dumps({"signature_name": "serving_default", "instances": img.tolist()})
+    data = json.dumps({"signature_name": "serving_default",
+                       "instances": img.tolist()})
     try:
         headers = {"content-type": "application/json"}
-        json_response = requests.post('http://localhost:8501/v1/models/fer2013:predict', data=data, headers=headers)
+        url = 'http://localhost:8501/v1/models/fer2013:predict'
```

Comment:
move it into constant as: BASE_URL,(http://localhost:8501/v1/) + FER_TFS_URL (models/fer2013:predict), then connect here it via urlparser. there are many constants used in this module . make changes accordingly